src/app.rs

```diff
@@ -1000,7 +1000,8 @@ This flag overrides -s/--case-sensitive and -S/--smart-case.
 fn flag_ignore_file(args: &mut Vec<RGArg>) {
     const SHORT: &str = "Specify additional ignore files.";
     const LONG: &str = long!("\
-Specify one or more files which contain ignore patterns. These patterns are
+Specifies a path to one or more files that specify a list of ignore rules, 
+of the same format as .ignore and .gitignore. These patterns are
```

Comment:
I guess this seems OK to me, but there is now a bit of repeated information. Could you revise the sentence starting with "Ignore patterns should be in the gitignore format ..."?

Reply:
Rephrased to remove duplication and bring gitignore format earlier. I hope this will help